app/assets/javascripts/discourse/app/components/table-header-toggle.js

```diff
@@ -10,15 +8,8 @@ export default Component.extend({
   labelKey: null,
   chevronIcon: null,
   columnIcon: null,
-
-  @discourseComputed("field", "labelKey")
-  title(field, labelKey) {
-    if (!labelKey) {
-      labelKey = `directory.${this.field}`;
-    }
-
-    return I18n.t(labelKey + "_long", { defaultValue: I18n.t(labelKey) });
-  },
+  translated: false,
+  onActiveRender: null,
```

Comment:
I think `headerDidRender` might be more descriptive here.

Reply:
That's not really true though. It's only on render, when the header is "active", meaning the table is being ordered by that header.

Reply:
Oh I see, that makes sense. Carry on!